tests/IResearch/IResearchInvertedIndexIteratorTest.cpp

```diff
@@ -135,7 +135,8 @@ class IResearchInvertedIndexIteratorTestBase
     _index = std::make_shared<arangodb::iresearch::IResearchInvertedIndex>(
         id, *_collection, std::move(meta));
     EXPECT_TRUE(_index);
-    EXPECT_TRUE(_index->init().ok());
+    bool pathExist = false;
```

Comment:
Suggest to verify `pathExist` value

Reply:
Why?

Reply:
Just to verify it works. Or  is there a test for that already added that I`ve missed?

Reply:
I think we don't need check something here. We use pathExist flag only init return failed result or throw exception

Reply:
But how then we make sure it works?

Reply:
Verify what we just take internal local variable, and make it output variable

We don't change any behaviour in these lines

Reply:
Maybe default initialization instead of `false` will be look better

Reply:
I mean just to check that it is true when it should be true and false otherwise. But indeed maybe in separate test.